module-2/backend/week_one.md

```diff
@@ -5,30 +5,69 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. List the five common HTTP verbs and what the purpose is of each verb.
+<ul>
+  <li>get - Reads info at specified path</li>
+  <li>put - updates or replaces info </li>
+  <li> post - updates or replaces info</li>
+  <li> delete - deletes info</li>
+</ul>
+
 2. What is Sinatra?
+<p>Its a domain specific languang that allows for web application development</p>
+
 4. What is MVC?
+<p>MVC stands for Model View Controller</p>
+
 5. Why do we follow conventions when creating our actions/path names in our Sinatra routes?
+<p>Because its RESTful?</p>

```

Comment:
Ha! Yes, but why bother with RESTful routes? I think the idea here is that if we follow these conventions that it allows us to get up to speed in new applications fairly quickly and provides users with a fairly familiar/expected experience when they look at links (e.g. if I follow a link to `http://www.some-server.com/donation/new` I'll be taken to a page to make a new donation).